statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -57,6 +59,9 @@ interface VariantManager {
             Variant(key = "zd", weight = 1.0, features = emptyList(), filterBy = { isEnglishLocale() }),
             Variant(key = "zg", weight = 1.0, features = listOf(ReturningUsersContinueWithoutPrivacyTips), filterBy = { isEnglishLocale() }),
             Variant(key = "zh", weight = 1.0, features = listOf(ReturningUsersSkipTutorial), filterBy = { isEnglishLocale() }),
+
+            // Tracking parameter removal - initial rollout
+            Variant(key = "my", weight = 1.0, features = listOf(VariantFeature.TrackingParameterRemoval), filterBy = { isEnglishLocale() })
```

Comment:
The feature is going to be rolled-out to 25% (It's not an experiment, it's just using the `VariantManager` to limit the rollout percentage of the feature). So if we remove the other variants before this is merged, then it would need to be set to 0.25.

Reply:
I've set the returning user variants to 0.0 and added two variants, 0.25 and 0.75 for the initial rollout.